test/optimizations/bulkcomm/block/exchange.chpl

```diff
@@ -70,15 +70,15 @@ proc main() {
     } else {
       const diags = getCommDiagnostics();
 
-      var gets : uint;
+      var gets : int;
       gets += + reduce diags.get;
       gets += + reduce diags.get_nb;
```

Comment:
Doesn't matter, but could probably write these as `var gets = + reduce (diags.get + diags.get_nb);` if we cared about not hardcoding the type (I can't imagine we're going to change it again though, so not important)

Reply:
(Just confirming, the 2 tests you fixed were the only ones I saw that were hardcoding the type)